src/views/TheHome.vue

```diff
@@ -1,28 +1,35 @@
 <template>
   <section>
-    <img class='puppy-picture' alt='picture of two puppies in the snow' src='https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSrnUwJ9xC5z-HtpYQZc3jHYJfcMtQUf8qzVg&usqp=CAU'/> 
+    <img
+      class="puppy-picture"
+      alt="picture of two puppies in the snow"
+      src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSrnUwJ9xC5z-HtpYQZc3jHYJfcMtQUf8qzVg&usqp=CAU"
+    />
     <h2>Find your pup a play date at the best dog parks around!</h2>
-    <button class='button-get-started'>Get Started - woof</button>
-    <!-- onClick - take user to search component, TBD -->
+    <FindPark />
   </section>
 </template>
 
 <script>
+import FindPark from './FindPark.vue';
 
+export default {
+  components: { FindPark },
+};
 </script>
 
 <style scoped>
-  section {
-    margin: 1em;
-    background-color: #AFECEB;
-    padding: 2em;
-  }
-  .puppy-picture {
-    width: 15em;
-    border-radius: 5%;
-  }
-  .button-get-started {
-    border-radius: 20%;
-    padding: 1em;
-  }
-</style>
\ No newline at end of file
+section {
+  margin: 1em;
+  background-color: #afeceb;
+  padding: 2em;
+}
+.puppy-picture {
+  width: 15em;
+  border-radius: 5%;
+}
+.button-get-started {
```

Comment:
We can delete this `.button-get-started` CSS since the button has been removed.

Reply:
Done! Thanks for catching this. I left it to copy/paste, but forgot to delete when I was done.